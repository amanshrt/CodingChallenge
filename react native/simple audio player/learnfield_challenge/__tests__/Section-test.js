import React from 'react';
import renderer from 'react-test-renderer';
import Section from '../src/Components/Section/index';

test('renders correctly', () => {
  const tree = renderer.create(<Section title="Step One" />).toJSON();
  expect(tree).toMatchSnapshot();
});
