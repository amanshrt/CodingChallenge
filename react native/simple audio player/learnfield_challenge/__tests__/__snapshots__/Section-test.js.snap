// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 32,
      "paddingHorizontal": 24,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 24,
          "fontWeight": "600",
        },
        Object {
          "color": "#000",
        },
      ]
    }
  >
    Step One
  </Text>
  <Text
    style={
      Array [
        Object {
          "fontSize": 18,
          "fontWeight": "400",
          "marginTop": 8,
        },
        Object {
          "color": "#444",
        },
      ]
    }
  />
</View>
`;
